command/acl_binding_rule_create.go

```diff
@@ -53,7 +53,7 @@ Create Options:
 
   -bind-type
     Specifies adjusts how this binding rule is applied at login time to internal
-    Nomad objects. Valid options are "role" and "policy".
+    Nomad objects. Valid options are "role", "policy", or "management".
 
   -bind-name
```

Comment:
Should the `-bind-name` help message and docs state that it must not be used if `-bind-type=management`?